### PR TITLE
NGI0 - Updating licensing aspects according REUSE specs v3

### PR DESCRIPTION
**Pull request by lucaslasota**
Hello, 

I am updating the licensing aspects of the repository according to the REUSE specifications v3[1].

Features changed:

- A LICENSES directory created 
- The text of LGPL-3.0-or-later added to the LICENSES directory
- Added SPDX comment headers 
- Fixed license information. Please note, however, that in the file `pwdsphinx/websphinx.py` it is not clear who is the copyrighter holder. 

Please feel free to adopt these recommendations if you find them convenient. 

Thanks,

Lucas

[1] https://reuse.software/

### CODE REVIEW

File: LICENSES/LGPL-3.0-or-later.txt [hunk 1]
```diff
@@ -0,0 +1,55 @@
+GNU LESSER GENERAL PUBLIC LICENSE
```
Comment (stef) [inline LICENSES/LGPL-3.0-or-later.txt:1]: why LGPL? i prefer GPL.

**Comment by stef** [comment]
wow. very nice. thank you for this. although i wonder why change to LGPL from GPL?

File: LICENSES/LGPL-3.0-or-later.txt (hunk 1 above)
Comment (lucaslasota) [reply LICENSES/LGPL-3.0-or-later.txt:1]: Hi Stefan, 
in the file `pwdsphinx/sphinxlib.py ` you indicate that the license is LGPL:
pitchforked sphinx is free software: you can redistribute it and/or modify it under the terms of the **GNU Lesser General Public License** as published by the Free Software Foundation
If it is not the case, and you will put the code under GPL v3, i can edit the pull request

Comment (stef) [reply LICENSES/LGPL-3.0-or-later.txt:1]: oh, that is my mistake then. i'm quite sure i wanted gpl.

Comment (lucaslasota) [reply LICENSES/LGPL-3.0-or-later.txt]: Changed license information to GPL v3 in all respective files

File: README.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+
+<!--
+SPDX-FileCopyrightText: 2018, Marsiske Stefan 
+
+SPDX-License-Identifier: CC-BY-SA-4.0
```
Comment (lucaslasota) [inline README.md:5]: Our license recommendation for documentation is the  CC-BY-SA-4.0. I added the license text to the directory. Feel free to adopt it. :)

File: pwdsphinx/websphinx.py [hunk 3]
```diff
@@ -1,22 +1,23 @@
 #!/usr/bin/env python3
 #
 # This file is part of WebSphinx.
-# SPDX-FileCopyrightText:  2018 pitchfork@ctrlc.hu
-# SPDX-FileCopyrightText:  GPL-2.0-or-later
```
Comment (lucaslasota) [inline pwdsphinx/websphinx.py]: As you mentioned that the preferred license in GPL v3, I update accordingly the comment header.

**Comment by stef** [comment]
wow, thank you so very much!
incredibly nice of you to make this effort!

File: README.md (hunk 2 above)
Comment (stef) [reply README.md:5]: good choice! i would have chosen this license

**Pull request merged**